helm/odigos/templates/odiglet/daemonset-discovery.yaml

```diff
@@ -0,0 +1,465 @@
+{{- if .Values.features.discoveryMode.enabled }}
```

Comment:
From now on we will have to maintain 2 template files for the odiglet.
can we share the same template and just modify the relevant parts in one file